community.md

```diff
@@ -67,6 +67,10 @@ Below is a list of projects and resources from the Blazor community (links in ea
 * [Blazor Bites](https://codedaze.io/tag/blazor-bites/) &ndash; Posts on ASP.NET Core and Blazor subjects.
 * [Learn Blazor](https://learn-blazor.com/) &ndash; Community documentation on Blazor.
 
+## Courses
+
+* [Blazor First Look Video Course](https://www.linkedin.com/learning/blazor-first-look)
```

Comment:
@Dedac Do you want the text to include "LinkedIn Learning"? Something like: "LinkedIn Learning: Blazor First Look".

Reply:
@danroth27 I don't think it needs to, I will update the name a bit though

Reply:
I'm just imagining that we will probably end up with a bunch of courses here. How would you want the LinkedIn course to be distinguished from the others?

Reply:
You are right, I went with it :)